internal/routes/all_routes.go

```diff
@@ -20,6 +21,7 @@ func pathPrefix() string {
 
 func apiRouter() http.Handler {
 	r := chi.NewRouter()
+	r.Use(identity.EnforceIdentity)
```

Comment:
So this middleware stores the identity in the context with key of different type. That is safe and the correct way to work with context, however, since fetching can be quite confusing, can you please make a helper in the ctxval package, we already have the function that returns logger. That could be quite useful already.

Reply:
There is a function for it in the identity package, but we can wrap that, if you feel it's useful :)